src/tests/screens/Home/survey.test.tsx

```diff
@@ -2,27 +2,122 @@ import React from 'react'
 
 import { render } from '@testing-library/react'
 
+import { SurveyBackgroundContext } from 'contexts/surveyBackground'
 import Survey from 'screens/Home/survey'
 
 describe('given survey page is mounted', () => {
-  it('renders survey header', () => {
-    const { getByTestId } = render(<Survey />)
-    const surveyHeader = getByTestId('surveyHeader')
+  describe('given valid survey list', () => {
+    it('renders survey header', () => {
+      const surveyList = [
+        {
+          node: {
+            id: '1',
+            title: 'title-1',
+            description: 'description-1',
+            coverImageUrl: 'image-1'
+          }
+        }
+      ]
+      const { getByTestId } = render(<Survey surveyList={surveyList} />)
+      const surveyHeader = getByTestId('surveyHeader')
 
-    expect(surveyHeader).toBeInTheDocument()
-  })
+      expect(surveyHeader).toBeInTheDocument()
+    })
+
+    it('renders survey time', () => {
+      const surveyList = [
+        {
+          node: {
+            id: '1',
+            title: 'title-1',
+            description: 'description-1',
+            coverImageUrl: 'image-1'
+          }
+        }
+      ]
+      const { getByTestId } = render(<Survey surveyList={surveyList} />)
+      const surveyTime = getByTestId('surveyTime')
+
+      expect(surveyTime).toBeInTheDocument()
+    })
+
+    it('renders surveyList', () => {
+      const surveyList = [
+        {
+          node: {
+            id: '1',
+            title: 'title-1',
+            description: 'description-1',
+            coverImageUrl: 'image-1'
+          }
+        }
+      ]
+      const { getByTestId } = render(<Survey surveyList={surveyList} />)
+      const surveyListPage = getByTestId('surveyList')
 
-  it('renders survey time', () => {
-    const { getByTestId } = render(<Survey />)
-    const surveyTime = getByTestId('surveyTime')
+      expect(surveyListPage).toBeInTheDocument()
+    })
 
-    expect(surveyTime).toBeInTheDocument()
+    it('renders correct survey background', () => {
+      const coverImageUrl = 'image-1'
+      const surveyList = [
+        {
+          node: {
+            id: '1',
+            title: 'title-1',
+            description: 'description-1',
+            coverImageUrl: coverImageUrl
+          }
+        }
+      ]
+      const state = { currentBackground: coverImageUrl }
+      const dispatch = () => null
+
+      const { getByTestId } = render(
+        <SurveyBackgroundContext.Provider value={{ state, dispatch }}>
+          <Survey surveyList={surveyList} />
+        </SurveyBackgroundContext.Provider>
+      )
+      const surveyPage = getByTestId('survey')
+
+      expect(surveyPage).toHaveStyle('background-image: url(image-1)')
+    })
   })
 
-  it('renders blank survey', () => {
-    const { getByTestId } = render(<Survey />)
-    const blankSurvey = getByTestId('blankSurvey')
+  describe('given empty survey list', () => {
```

Comment:
I add new tests to this file because header, time and background are part of survey page instead of blank survey.
Should I still move new test you mentioned to `blankSurvey.test.tsx`?